autoload/ollama/setup.vim

```diff
@@ -412,21 +421,18 @@ use_venv = vim.eval('g:ollama_use_venv') or 0
 
 # Should we use a venv?
 if use_venv:
-    # Create default venv path
     venv_path = os.path.join(os.environ['HOME'], '.vim', 'venv', 'ollama')
-    # Check if the venv path exists
     if os.path.exists(venv_path):
-        #print('Found venv:', venv_path)
-
         venv_bin = os.path.join(venv_path, 'bin', 'python3')
-        venv_site_packages = os.path.join(venv_path, 'lib', f'python{sys.version_info.major}.{sys.version_info.minor}', 'site-packages')
-
-        # Ensure the virtual environment's site-packages is in sys.path
+        venv_site_packages = os.path.join(
+            venv_path,
+            'lib', f'python{sys.version_info.major}.{sys.version_info.minor}',
+            'site-packages'
+        )
```

Comment:
*minor*

If you are switching to vertical formatting, maybe format all the argument vertically:

```suggestion
        venv_site_packages = os.path.join(
            venv_path,
            'lib',
            f'python{sys.version_info.major}.{sys.version_info.minor}',
            'site-packages'
        )
```